Makefile

```diff
@@ -215,6 +215,18 @@ rcc:
 	./ui/generate-rcc.sh
 	rcc --binary ui/resources.qrc -o ./resources.rcc
 
+# default token is a free-tier token with limited capabilities and usage
+# limits; our docs should include directions for community contributor to setup
+# their own Infura account and token instead of relying on this default token
+# during development
+DEFAULT_TOKEN := df3bd604ac4b4073823e72f6617fc7db
+INFURA_TOKEN ?= $(DEFAULT_TOKEN)
+ifeq ($(INFURA_TOKEN),)
+ # there should always be a token defined at compile time
+ override INFURA_TOKEN = $(DEFAULT_TOKEN)
+endif
```

Comment:
This `ifeq` is unnecessary. The `?=` already does the same thing.

Reply:
There override is there in case of e.g. `make INFURA_TOKEN= nim_status_client` thereby effectively supplying the empty string as the INFURA_TOKEN since `?=` will honor the blank RHS of `INFURA_TOKEN=`. I ran into that kind of thing when writing a bash script that invokes make and had a typo where an environment variable was supposed to fill in the RHS of `=`.

Reply:
In light of https://github.com/status-im/nim-status-client/pull/1261#discussion_r515345122 I got rid of it.